userguide/system.rst

```diff
@@ -822,6 +818,108 @@ with locking enabled:
        LockOnReset:     1
 
 
+.. index:: SED, SED Password
+.. _Managing SED Password and Data:
+
+Managing SED Password and Data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This section contains command line instructions to manage SED
+passwords and data. The command used is
+`sedutil-cli(8) <https://www.mankier.com/8/sedutil-cli>`__. Most
+SEDs are TCG-E (Enterprise) or TCG-Opal
+(`Opal v2.0 <https://trustedcomputinggroup.org/wp-content/uploads/TCG_Storage-Opal_SSC_v2.01_rev1.00.pdf>`__).
+
+.. warning:: These commands can be descructive to data and passwords.
+   Take necessary data backup procedures and use the commands with
+   caution.
+
+Drives can be checked individually to determine if it is a SED and what
+variant of SED it is with this comand:
+
+.. code-block:: none
+
+   root@tn01b:~ # sedutil-cli --isValidSED /dev/da0
+   /dev/da0 SED --E--- Micron_5N/A U402
+
+
+To check all disks at once, run this command:
+
+.. code-block:: none
+
+   root@tn01b:~ # sedutil-cli --scan
+   Scanning for Opal compliant disks
+   /dev/ada0 No 32GB SATA Flash Drive SFDK003L
+   /dev/ada1 No 32GB SATA Flash Drive SFDK003L
+   /dev/da0 E Micron_5N/A U402
+   /dev/da1 E Micron_5N/A U402
+   /dev/da12 E SEAGATE XS3840TE70014 0103
+   /dev/da13 E SEAGATE XS3840TE70014 0103
+   /dev/da14 E SEAGATE XS3840TE70014 0103
+   /dev/da2 E Micron_5N/A U402
+   /dev/da3 E Micron_5N/A U402
+   /dev/da4 E Micron_5N/A U402
+   /dev/da5 E Micron_5N/A U402
+   /dev/da6 E Micron_5N/A U402
+   /dev/da9 E Micron_5N/A U402
+   No more disks present ending scan
+   root@tn01b:~ #
+
+
+In the commands above, "E" represents that it is a TCG-E (Enterprise)
```

Comment:
This would be easier to read either as a table.
 Table formatting will have to be fixed, since
Github isn't helping.
```
The value in the second column of output indicates the type of SED:

+-------------------------------------+
| :literal:`E`  | TCG-E (Enterprise) |
| :literal:`2`  | TCG-Opal (Opal v2.0) |
| :literal:`No` | Not a SED |
+-------------------------------------+
```